client/stripe-utils/utils.js

```diff
@@ -22,7 +23,11 @@ import { errorTypes, errorCodes } from './constants';
  * @return  {StripeServerData} Stripe server data.
  */
 const getStripeServerData = () => {
-	const stripeServerData = getSetting( 'stripe_data', null );
+	// Classic checkout or blocks-based one.
+	const stripeServerData =
+		'undefined' !== typeof wc_stripe_upe_params
+			? wc_stripe_upe_params
+			: wc.wcSettings.getSetting( 'stripe_data', null );
```

Comment:
I'm not sure I understand the change proposed... the ternary if is checking if `wc_stripe_upe_params` is defined (`'undefined' !== typeof wc_stripe_upe_params`), and if it is, it will be assigned to `stripeServerData`, and if not `wc.wcSettings.getSetting( 'stripe_data', null )` will be used instead:

```
const stripeServerData = 'undefined' !== typeof wc_stripe_upe_params
		         ? wc_stripe_upe_params
		         : wc.wcSettings.getSetting( 'stripe_data', null );
```

It's the same as in WC payments:

https://github.com/Automattic/woocommerce-payments/blob/a19aafbcbdb63e10921c7355b1858fd047969290/client/utils/checkout.js#L9-L17

Reply:
Oof, I'm so sorry, you're right. I just read the code here wrong, nevermind this comment! 🤦‍♂️